src/Components/SearchBar.js

```diff
@@ -83,12 +83,15 @@ if (Privacy.isAllowed(PRIVACY_ACTIONS.SEARCH)) {
                         suggestion:
                             this.props.suggestion_template ||
                             function (suggestion) {
```

Comment:
Since you use `function` instead of arrow functions `() => {}`, the context is not correctly bound to the function (I know, I didn't expect there to be any semantic difference there either, when I first encountered this. That's JavaScript for you). That is why the test fails. Please use an arrow function to fix this.